src/Hedgehog/paket.template

```diff
@@ -2,6 +2,8 @@ type
     project
 id
     Hedgehog
+version
+	0.7.0.0
```

Comment:
Do we even need `paket.template` now? :sweat_smile:

Reply:
At the moment, yes. Building from the `.fsproj` with `dotnet` tooling generates a package with different version bounds...